avalon/tools/loader/model.py

```diff
@@ -206,23 +210,23 @@ def data(self, index, role):
             return
 
         if role == QtCore.Qt.DisplayRole:
-            if index.column() == 1:
+            if index.column() == self.Columns.index("family"):
```

Comment:
Be aware that `data()` is called very very very often. `Column.index()` on a list is relatively slow. I like the increased readability but maybe it's worth it to have a lookup table that is initialized on `__init__` only once?

```python
self.columns_index = dict((key, i) for i in enumerate(self.Columns))
```

Then this line just becomes:
```python
self.columns_index["family"]
```

---

Of course this would be for all of the `self.Columns.index()` lines.